### ISSUE TITLE
checkTypes.assert.array.of.string() throwing wrong exception

### ISSUE THREAD

**Issue by stefanheine** [philbooth/check-types.js#77]
I found a regression in 7.1.2:

```
const checkTypes = require('check-types');

checkTypes.assert.array.of.string('string', 'not an array of string');
```

I have expected this to throw 'not an array of string', but in fact it's throwing 'string'. this had been working as expected in 7.0.1.

**Comment by philbooth** [comment]
Thanks for reporting this! I believe it's the same problem as #75, I'll look into it this evening.

**Comment by philbooth** [comment]
Apologies again for this, it should be fixed in version `7.1.3`, published just now. Let me know if you get any problems!

**Comment by stefanheine** [comment]
thanks. looks ok.
